### ISSUE TITLE
Support for ARM64 WoA images

### ISSUE THREAD

**Issue by dege-diosg** [upx/upx#502]
## What's the problem (or question)?
This is more like a feature request, but are you planning adding support for ARM64 WoA (Win32) images?

**Comment by jreiser** [comment]
Is "WoA image" a compiled binary executable file in PE format?  Please upload ("Attach files by ..."), give the URL, or email an actual example.

**Comment by dege-diosg** [comment]
Yes, they are the ARM32/ARM64 version of the regular PE images (WoA == Windows on ARM).

DosBox-X has builds of both ARM32 and ARM64 for Windows:
https://github.com/joncampbell123/dosbox-x/blob/master/INSTALL.md#windows-packages-installer-or-portable

https://github.com/joncampbell123/dosbox-x/releases/download/dosbox-x-v0.83.13/dosbox-x-vsbuild-arm32-20210501000322.zip
https://github.com/joncampbell123/dosbox-x/releases/download/dosbox-x-v0.83.13/dosbox-x-vsbuild-arm64-20210501000322.zip

**Comment by jreiser** [comment]
@dege-diosg Yes, please upload ("Attach files by ..."), give the URL, or email a small actual compiled binary example.  Use zip/gzip to compress the transfer.

**Comment by dege-diosg** [comment]
Here are 2 dll's:

http://dege.fw.hu/temp/dosbox_plugins_arm64.zip

Copy them into the folder of dosbox-x.exe. D3D9 is the default that's used for dosbox video output so if dosbox starts and you can use it as usual then the plugins work.

**Comment by markus-oberhumer** [comment]
Closing in favor of #650
